src/content/getusermedia/gum/js/test.js

```diff
@@ -12,44 +12,22 @@
 /* jshint node: true */
 var test = require('tape');
 
-// https://code.google.com/p/selenium/wiki/WebDriverJs
 var webdriver = require('selenium-webdriver');
-var chrome = require('selenium-webdriver/chrome');
-var firefox = require('selenium-webdriver/firefox');
-
-var profile = null;
-var firefoxOptions = null;
-var chromeOptions = null;
-
-// Firefox options.
-// http://selenium.googlecode.com/git/docs/api/javascript/module_selenium-webdriver_firefox.html
-var profile = new firefox.Profile();
-profile.setPreference('media.navigator.streams.fake', true);
-var firefoxOptions = new firefox.Options()
-    .setProfile(profile)
-    .setBinary('node_modules/.bin/start-firefox');
-
-// Chrome options.
-// http://selenium.googlecode.com/git/docs/api/javascript/module_selenium-webdriver_chrome_class_Options.html#addArguments
-var chromeOptions = new chrome.Options()
-    .setChromeBinaryPath('node_modules/.bin/start-chrome')
-    .addArguments('allow-file-access-from-files')
-    .addArguments('use-fake-device-for-media-stream')
-    .addArguments('use-fake-ui-for-media-stream');
+var seleniumHelpers = require('../../../../../test/selenium-lib');
 
 test('Video width and video height are set on GUM sample', function(t) {
   // FIXME: use env[SELENIUM_BROWSER] instead?
-  var driver = new webdriver.Builder()
-      .forBrowser(process.env.BROWSER)
-      .setFirefoxOptions(firefoxOptions)
-      .setChromeOptions(chromeOptions)
-      .build();
+  var driver = seleniumHelpers.buildDriver();
 
   driver.get('file://' + process.cwd() +
       '/src/content/getusermedia/gum/index.html')
   .then(function() {
     t.pass('Page loaded');
   })
+  .then(function() {
+    // sleep for one second
+    return driver.sleep(1000);
```

Comment:
In that case is 1 second enough? With debug builds + camera hardware state, I've used 4 seconds in some of my tests to be sure, but that was with real hardware, rather than fake, so maybe that's ok.